libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -513,18 +520,18 @@ namespace SohImGui {
 
             ImGui::Text("Platform: Windows");
             ImGui::Text("Status: %.3f ms/frame (%.1f FPS)", 1000.0f / framerate, framerate);
-            if (UseInternalRes()) {
-                ImGui::Text("Internal Resolution:");
-                ImGui::SliderInt("Mul", reinterpret_cast<int*>(&gfx_current_dimensions.internal_mul), 1, 8);
-            }
+            ImGui::Text("Internal Resolution:");
```

Comment:
This was my fault, but it would be cool if you move this into a Graphics category on the menubar